### PR TITLE
#101: Refactor tests to use new createWrapper

### PR DESCRIPTION
**Pull request by roryschadler**
fix(#101): Refactor tests to use new createWrapper

Refactored tests to remove repeated setup code and use new consistent
`createWrapper`. Also suppressed `console.log` calls in howto tests

### CODE REVIEW

File: app/tests/unit/pages/nanomine/howto.spec.js [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+  // Keep native behavior for other methods
+  error: console.error,
+  warn: console.warn,
+  info: console.info,
+  debug: console.debug
+}
 
 describe('HowTo.vue', () => {
+  beforeAll(async () => {
+    wrapper = createWrapper(HowTo, { props: {}, slots: {} })
```
Comment (tholulomo) [inline app/tests/unit/pages/nanomine/howto.spec.js:20]: I will suggest if the props and slots are empty, we can throw in an empty object like so: 
`createWrapper(HowTo, {})`

**Review by tholulomo — APPROVED** [review]
👍

**Comment by kevinzunigacuellar** [comment]
Maybe we can add a no console eslint rule to prevent console logs to the main dev branch

**Review by stouffers — APPROVED** [review]

File: app/tests/unit/pages/nanomine/howto.spec.js (hunk 1 above)
Comment (roryschadler) [reply app/tests/unit/pages/nanomine/howto.spec.js]: Done in ca97924

**Pull request merged**